OutOfSchool/OutOfSchool.WebApi/Controllers/V1/SubordinationStructure/InstitutionHierarchyController.cs

```diff
@@ -14,7 +17,7 @@ namespace OutOfSchool.WebApi.Controllers.V1.SubordinationStructure
     [ApiController]
     [ApiVersion("1.0")]
     [Route("api/v{version:apiVersion}/[controller]/[action]")]
-    [HasPermission(Permissions.SystemManagement)]
+    [Authorize]
```

Comment:
it's either we have a certain permission or leave it for anonymous because just having an account in the application is not secure enough :). It's ok if actions need different permission (get is anonymous, create is admin only, etc.)

Reply:
Ok ;)